Library/Formula/tools-osx.rb

```diff
@@ -0,0 +1,20 @@
+class ToolsOsx < Formula
+  homepage "https://github.com/morgant/tools-osx"
+  url "https://github.com/morgant/tools-osx/archive/trash-0.5.2.tar.gz"
+  sha1 "b2afec002c0f4c56e188ff8a020e7241b5919fb3"
+
+  conflicts_with "osxutils", :because => "both install a trash binary"
+  conflicts_with "trash", :because => "both install a trash binary"
+
+  def install
+    rake
+    libexec.install "bin"
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system "/usr/bin/touch", "testfile"
```

Comment:
You can just do ` touch "testfile" ` here. No need to invoke the system for this line :).